examples/minitest_helper.rb

```diff
@@ -35,4 +35,5 @@ def use_natural_assertions_if_supported(enabled=true)
 end
 
 Minitest::Spec.send(:include, GivenAssertions)
+Minitest::Test.send(:include, GivenAssertions)
```

Comment:
why are these two lines in the examples? is this something users actually have to do on their end? thought that was auto mixed-in by just requiring the gem?